app/src/main/java/com/indoornavigation/CanvasView.java

```diff
@@ -0,0 +1,12 @@
+package com.indoornavigation;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+
+public abstract class CanvasView extends ViewGroup
```

Comment:
Ta klasa nie jest do niczego potrzebna. Klasa map moze dziedziczyc po ViewGroupie, a tak wlasciwie to powinna dziedziczyc po View